src/Products/Metadata/Controllers/MetadataApiController.cs

```diff
@@ -139,13 +136,20 @@ public HttpResponseMessage loadProperties(PostedDataEntity postedData)
 
                 else if (filePropertyCategory == FilePropertyCategory.Default)
                 {
-                    // Fetch all metadata properties that fall into a particular category
-                    IEnumerable<MetadataProperty> defaultProperties = metadata.FindProperties(p => p.Value.RawValue != null
-                                                                                                && !string.Empty.Equals(p.Value.RawValue)
-                                                                                                && (!p.Value.RawValue.GetType().IsValueType
-                                                                                                    || !p.Value.RawValue.Equals(Activator.CreateInstance(p.Value.RawValue.GetType())))
-                                                                                                && p.Tags.Any(t => t.Category == Tags.Content));
-
+                    IEnumerable<MetadataProperty> defaultProperties = new List<MetadataProperty>();
```

Comment:
Codacy found an issue: [Remove this useless assignment to local variable 'defaultProperties'.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-MVC/pullRequest?prid=5989453)